game_event/views.py

```diff
@@ -130,7 +133,8 @@ def remove_from_event(request, id):
         return render(request, 'game_event/result.html', {'event_exists': False})
     player = Player.objects.get(user=request.user)
     GameEventPlayer.objects.get(game_event=event, player=player).delete()
-    return render(request, 'game_event/result.html', {'event_exists': True})
+    event_exists = event.delete_game_event()
```

Comment:
I either missing something with the naming or the logic. `delete_game_event` returns true if it deletes the event.  How come in that case we say that the event exists. Shouldn't it be the other way around?